### ISSUE TITLE
Plugin not working

### ISSUE THREAD

**Issue by digifost** [nstudio/nativescript-paytm#7]
### Which platform(s) does your issue occur on?
- Android
- iOS/Android versions 
- emulator or device. What type of device? - Emulator

### Please, provide the following version numbers that your issue occurs with:

- CLI: (run `tns --version` to fetch it) - 7.0.4
- Runtime(s): (look for the `"tns-android"` and `"tns-ios"` properties in the `package.json` file of your project) - 6.5.0

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.
I have integrated the plugin with Nativescript-angular for Android. However, after following the demo app, i am just getting the 404 error. 
Please let me know if i am missing anything

**Comment by shiv19** [comment]
Hi @digifost,
I wrote this plugin 3 years ago, so I think some paytm APIs might have changed by now.
If you follow through their integration docs and find any changes, please make those changes to a fork of this repo and PR is back here, so that we can publish a new version for you :)

**Comment by digifost** [comment]
@shiv19 ..Thanks for your quick response. Sorry to say, but I am very new to Nativescript, havent gone that far to create or access Native API's, can you help me on this ?

**Comment by shiv19** [comment]
@digifost 
actually, can you try integrating the plugin with nativescript-angular first?
Because the 404 error is expected as the demo merchant details that I used 3 years ago may no longer be valid.
Let me know if you still get 404 after integration.

**Comment by digifost** [comment]
@shiv19 ..I am trying with Nativescript-Angular only. And also i am using different merchant details and generating checksum from backend.
Just one quick question, do we get 404 error even when Order details are incorrect?

**Comment by shiv19** [comment]
Yes it is possible to get a 404 if the order details are incorrect.
For instance if the website is wrong, it could lead to a 404.

My timezone is NZST, will only be able to look at this later tomorrow.

**Comment by digifost** [comment]
Sure, I will wait for your response.
Till then i will also try and keep you posted.

Appreciate your help....

**Comment by shiv19** [comment]
Hey @digifost 
on a quick look at their developer docs, I noticed that they have completely changed the name of their sdk
![image](https://user-images.githubusercontent.com/9407019/92470127-16020200-f22a-11ea-95f3-e05602da8e3e.png)

so this plugin probably requires a rewrite at this point..

**Comment by Rashids5776** [comment]
any updates on this please?

**Comment by shiv19** [comment]
@Rashids5776 I don't intend to rewrite this plugin currently.
But feel free to take the code in this plugin as reference and write your own plugin for it :)
